providers/google/tests/deprecations_ignore.yml

```diff
@@ -15,17 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 ---
-
```

Comment:
Those deprecation ignores for Amazon are no longer needed as I found out. And we have now separate deprecations_ignore per provder, and the common autouse fixture will find and apply those deprecations ignore.